frontend/src/components/MetricCard/metricCard.css

```diff
@@ -10,6 +12,18 @@
   gap: 4px;
 }
 
+.card {
+  width: 309px;
+  
+  background: rgba(18, 7, 33, 1);
+  border: 1px solid rgba(54, 41, 78, 1);
```

Comment:
colors should be used from css variables

Reply:
I've made sure I did before pushing, would rectify.